plugins/wiggle/src/LinearWiggleDisplay/models/model.tsx

```diff
@@ -84,13 +84,12 @@ const stateModelFactory = (
     .volatile(() => ({
       statsReady: false,
       message: undefined as undefined | string,
-      stats: observable({ scoreMin: 0, scoreMax: 50 }),
+      stats: undefined as undefined | { scoreMin: number; scoreMax: number },
```

Comment:
don't use an initial value of 0-50, instead leave it undefined, and don't use observable data structure. just replace the entire stats object on changes, which works for the level of volatile observability